sigmf-spec.md

```diff
@@ -557,21 +559,47 @@ deciding which open license fits your needs best. Cornell University has also
 created [a guide](https://data.research.cornell.edu/content/intellectual-property#data-licensing) 
 to help you make these choices.
 
-## SigMF Compliance by Applications
+## SigMF Compliance
 
-In order to be `SigMF Compliant`, an application must meet the following
+The term 'SigMF Compliant' is used throughout this document; there are two
+aspects to SigMF compliance. In order for an application or dataset to say it is
+'SigMF Compliant', specific conditions MUST be met, outlined in the following
+sections. Provided the below criteria are met, an application or dataset may
+indicate that it is 'SigMF Compliant'.
+
+### SigMF Compliant Datasets
+
+In order to be 'SigMF Compliant', an dataset must meet the following
+requirements:
+
+1. Adheres to and supports the file rules, dataset formats, `objects`,
+   `namespaces`, and `names` specified by this document.
+1. MUST contain all REQUIRED fields with the correct datatype listed the `core`
+   namespace, and any namespace listed in the `extensions` array.
+1. MUST NOT contain fields that are not outlined in the `core` or a listed
+   `extensions` namespace.
+
+Non Compliant Datasets MAY work with SigMF Compliant applications, but this is
```

Comment:
@bhilburn Hrmmmm.... addressing your feedback; does this need to be changed to "non compliant recordings" [all over] then?